### DISCUSSION TITLE
Something went wrong with LinkedIn or Supabase?

### DISCUSSION THREAD

**Discussion by Edier-Gama** [supabase/supabase#22708] [Questions] [answered]
I'm developing a website with LinkedIn OIDC integration. Last night, on 11/04, I went to sleep, and the next day, I couldn't log in with LinkedIn, even though I didn't modify anything

I created a new Supabase project and updated the credentials in my project, but it still doesn't work

I had this error:

_**http://localhost:3000/auth/auth-code-error#error=s…Error+getting+user+profile+from+external+provider**_

My code has been working excellently for weeks, but suddenly it's not functioning properly. Does anyone know why this might be happening? **I am using Next.js 14 with App Router and the new supabase/ssr**

**Comment by GaryAustin1** [comment, marked as answer]
https://github.com/supabase/auth/pull/1534  A user reported the issue and seems to be suggesting there is a fix that needs to be done.

**Reply by kangmingtay** [reply]
hey everyone, seems like this was a breaking change that wasn't being communicated on linkedin's end :( - we've made a [fix](https://github.com/supabase/auth/pull/1536) for it but it will take awhile before we are able to roll it out to everyone on the hosted platform 

if you need this urgently, please open a ticket at https://supabase.help

**Reply by lalitntaparia** [reply]
@kangmingtay Issue intermittently persist.

**Reply by Edier-Gama** [reply]
> @kangmingtay Issue intermittently persist.

Hi everyone, @kangmingtay in my case everything is working fine after the fix. I recommend opening a new discussion and including details of your code @lalitntaparia
